io-engine/src/core/snapshot.rs

```diff
@@ -113,6 +146,9 @@ pub trait SnapshotOps {
 
     /// List Snapshots.
     fn list_snapshot(&self) -> Vec<VolumeSnapshotDescriptor>;
+
+    /// List Single snapshot details based on UUID.
+    fn list_single_snapshot(&self) -> Vec<VolumeSnapshotDescriptor>;
```

Comment:
nit: consider renaming list_snapshot to list_snapshots, and list_single_snapshot to list_snapshot

Reply:
modified the name.